app/Http/Controllers/UsersController.php

```diff
@@ -7,14 +7,27 @@
 
 class UsersController extends Controller
 {
+     /**
+     * Instantiate a new controller instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        $this->middleware('auth');
+    }
+
     /**
      * Display a listing of the resource.
      *
      * @return \Illuminate\Http\Response
      */
     public function index()
     {
-        //
+        //exclude user currently logged in
+        $users = User::where('id','!=',auth()->user()->id)->paginate(10);
```

Comment:
can you express this line of code in the model part since this already dealing with queries i think it's better to distinguish the work of controller and model. In this part this will be the model part accessing the data from database. you can refer to what justin did.